Index: os-pylint-restricted-imports/tests/test_imports.py
===================================================================
--- os-pylint-restricted-imports.orig/tests/test_imports.py
+++ os-pylint-restricted-imports/tests/test_imports.py
@@ -3,7 +3,7 @@ from unittest import mock
 
 import astroid
 
-from pylint.testutils import CheckerTestCase, Message
+from pylint.testutils import CheckerTestCase, MessageTest
 
 from pylint_restricted_imports import RestrictedImportChecker
 
@@ -41,7 +41,8 @@ class TestRestrictedImportChecker(Checke
         with mock.patch.object(self.checker, "_import_module") as mock_import:
             mock_import.return_value = astroid.Module("bar", "bar")
             with self.assertAddsMessages(
-                Message("restricted-import", node=node, args=("bar", "bar"))
+                MessageTest("restricted-import", node=node, args=("bar", "bar"),
+                            line=1, end_line=1, col_offset=0, end_col_offset=10)
             ):
                 self.checker.visit_import(node)
 
@@ -52,7 +53,8 @@ class TestRestrictedImportChecker(Checke
         with mock.patch.object(self.checker, "_import_module") as mock_import:
             mock_import.return_value = astroid.Module("bar.wibble", "bar.wibble")
             with self.assertAddsMessages(
-                  Message("restricted-import", node=node, args=("bar.wibble", "bar"))
+                  MessageTest("restricted-import", node=node, args=("bar.wibble", "bar"),
+                              line=1, end_line=1, col_offset=0, end_col_offset=22)
             ):
                 self.checker.visit_importfrom(node)
 
@@ -70,7 +72,8 @@ class TestRestrictedImportChecker(Checke
                 astroid.Module("bar", "bar module"),
             ]
             with self.assertAddsMessages(
-                Message("restricted-transitive-import", node=node, args=("baz", "bar"))
+                MessageTest("restricted-transitive-import", node=node, args=("baz", "bar"),
+                            line=1, end_line=1, col_offset=0, end_col_offset=10)
             ):
                 self.checker.visit_import(node)
 
@@ -88,6 +91,7 @@ class TestRestrictedImportChecker(Checke
                 astroid.Module("bar", "bar module"),
             ]
             with self.assertAddsMessages(
-                Message("restricted-transitive-import", node=node, args=("baz.wibble", "bar"))
+                MessageTest("restricted-transitive-import", node=node, args=("baz.wibble", "bar"),
+                            line=1, end_line=1, col_offset=0, end_col_offset=22)
             ):
                 self.checker.visit_importfrom(node)
